viewer/settings.py

```diff
@@ -45,6 +51,7 @@
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'dockerflow.django.middleware.DockerflowMiddleware',
```

Comment:
You may want to move this ahead of the other middlewares so that it short-circuits them. Unless you need session and/or auth info in the dockerflow checks, which I don't think you do.